CHANGES.rst

```diff
@@ -10,6 +10,78 @@
 
 .. towncrier release notes start
 
+3.8.6 (2023-10-07)
+==================
+
+Security bugfixes
+-----------------
+
+- Upgraded the vendored copy of llhttp_ to v9.1.3 -- by :user:`Dreamsorcerer`
+
+  Thanks to :user:`kenballus` for reporting this, see
+  https://github.com/aio-libs/aiohttp/security/advisories/GHSA-pjjw-qhg8-p2p9.
+
+  .. _llhttp: https://llhttp.org
+
+  `#7647 <https://github.com/aio-libs/aiohttp/issues/7647>`_
+
+- Updated Python parser to comply with RFCs 9110/9112 -- by :user:`Dreamorcerer`
+
+  Thanks to :user:`kenballus` for reporting this, see
+  https://github.com/aio-libs/aiohttp/security/advisories/GHSA-gfw2-4jvh-wgfg.
+
+  `#7663 <https://github.com/aio-libs/aiohttp/issues/7663>`_
+
+
+Deprecation
+-----------
+
+- Added `fallback_charset_resolver` parameter in `ClientSession` to allow a user-supplied
+  character set detection function.
+
+  Character set detection will no longer be included in 3.9 as a default. If this feature is needed,
+  please use `fallback_charset_resolver <https://docs.aiohttp.org/en/stable/client_advanced.html#character-set-detection>`_.
```

Comment:
Can this be linked using the `ref` role?

Reply:
Not sure it'd work outside of Sphinx? e.g. when appearing in Github changelog.

Reply:
In fact, if you look at changelog for 3.8.2, I think several of your suggestions actually mess up rendering:

![image](https://github.com/aio-libs/aiohttp/assets/2203121/c9504d95-3c8f-4723-aaec-950596380412)

Reply:
I've updated the ones that seems like they'll render correctly, and ignored the other suggestions.

Reply:
I decided to ignore that long ago. You could add some post-processing but it's probably not worth it.